src/transformers/models/fsner/configuration_fsner.py

```diff
@@ -0,0 +1,126 @@
+# coding=utf-8
+# Copyright sayef and The HuggingFace Inc. team. All rights reserved.
```

Comment:
The copyrights here are reserved for the organizations from which some of the code was taken - if I understand correctly FSNER was originally designed by Microsoft, so I would put Microsoft here instead of `sayef`